packages/oracle-manager/scripts/deploy.js

```diff
@@ -0,0 +1,90 @@
+const hre = require('hardhat');
+const {
+  TASK_DEPLOY,
+  SUBTASK_GET_MULTICALL_ABI,
+  SUBTASK_GET_DEPLOYMENT_INFO,
+} = require('@synthetixio/hardhat-router/task-names');
+
+/**
+ * Generate the file contracts/Router.sol including the given modules in its source.
+ */
+module.exports.deploy = async function deploy(runtime, prefix, modules) {
```

Comment:
ideally (don't worry about for this PR but just want to make a note) this deployment file should be embedded within the `hardhat-router` or something in order to make it easy for any project to just call cannon run step here without needing to copy and paste it like this